scripts/delayed.js

```diff
@@ -1,7 +1,9 @@
 // eslint-disable-next-line import/no-cycle
 import { sampleRUM, isInternalPage } from './lib-franklin.js';
 // eslint-disable-next-line import/no-cycle
-import { getEnvType, loadConsentManager, loadScript } from './scripts.js';
+import {
```

Comment:
I guess this file hasn't really changed so it doesn't need to be part of the PR?